python/src/JansenSensitivityAlgorithm_doc.i.in

```diff
@@ -38,7 +38,7 @@ The [jansen1999]_ method is used to estimate both first
 and total order indices.
 
 **The class constructor** ``JansenSensitivityAlgorithm(inputDesign, outputDesign, N)``
-**requires an** ``outputDesign`` **produced by** :class:`~openturns.SobolIndicesExperiment`
+**requires an** ``inputDesign`` **produced by** :class:`~openturns.SobolIndicesExperiment`
```

Comment:
actually the inputDesign values are not used, only the output values are, currently the input design values are asked so that we can check with size/dimension that the output size matches

Reply:
How do you think this sentence should be formulated?

Reply:
it was ok before ^^

Reply:
IDK, the current phrasing could be interpreted as "outputDesign should be the output of SobolIndicesExperiment", which is wrong. Would this alternative be better?
```suggestion
**requires an** ``outputDesign`` **matching an ``inputDesign`` produced by** :class:`~openturns.SobolIndicesExperiment`
```

Reply:
ok

Reply:
Something like :
```
The class constructor JansenSensitivityAlgorithm(inputDesign, outputDesign, N) requires a specific structure for outputDesign, and therefore for inputDesign. This last one should be generated using SobolIndicesExperiment (see example below). Otherwise, results will be worthless.
```

Reply:
Yes, this is even better. I'll commit this when I get around to it.